modules/lightning_features/lightning_core/lightning_core.module

```diff
@@ -81,6 +110,50 @@ function lightning_core_disable_buttons(array $element) {
 }
 
 /**
+ * Implements hook_form_alter().
+ */
+function lightning_core_form_alter(array &$form, FormStateInterface $form_state) {
+  $form_object = $form_state->getFormObject();
+
+  if ($form_object instanceof EntityFormInterface) {
+    $entity = $form_object->getEntity();
+
+    if ($entity instanceof EntityDisplayModeInterface || $entity instanceof RoleInterface) {
+      $form['description'] = [
+        '#type' => 'textarea',
```

Comment:
The description looks kind of funny bumped up against the rest of the help text with no paragraph break. What if we changed this to a textfield instead of a text area and wrapped the output in a p tag when it's rendered with the help. A textfield is also more conducive to being used as a description for the select lists in panelizer default choices if and when we implement that.

FWIW, core vocabulary description fields are also textfields not textareas.